Resources/Prototypes/_Sunrise/Entities/Objects/Weapons/Guns/Projectiles/projectiles.yml

```
@@ -75,6 +75,24 @@
     reflective:
     - Energy
 
+- type: entity
+  id: BulletGrenadeFragLight
+  name: Light frag grenade
+  parent: BaseBulletGrenade
+  categories: [ HideSpawnMenu ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)